interfaces/datastructure/PointCloud.h

```diff
@@ -41,8 +41,8 @@ class  SOLARFRAMEWORK_API PointCloud : public Lockable {
 
 	///
 	/// @brief PointCloud constructor.
-	///
-	PointCloud() { m_id = 0; };
+    ///
+    PointCloud() { m_id = 0; m_descriptorType = DescriptorType::AKAZE; };
```

Comment:
nit: use member initializer, and remove those ([source](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c48-prefer-default-member-initializers-to-member-initializers-in-constructors-for-constant-initializers))

```cpp
    SolAR::datastructure::DescriptorType  m_descriptorType{DescriptorType::AKAZE};
    uint32_t m_id{0};
```